packages/charts/react-charting/package.json

```diff
@@ -15,12 +15,9 @@
   "license": "MIT",
   "scripts": {
     "build": "just-scripts build",
-    "lint": "just-scripts lint",
     "test": "cross-env TZ=UTC just-scripts test",
-    "just": "just-scripts",
```

Comment:
these targets are not run/used anywhere - removing dupes created by nx inferred plugin